common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -277,6 +279,26 @@ def student_view(self, context):
         prereq_met = True
         prereq_meta_info = {}
 
+        # Content type gating for FBE previously only gated individual blocks
+        # This was an issue because audit learners could start a timed exam and then be unable to complete the exam
+        # even if they later upgrade because the timer would have expired.
+        # For this reason we check if content gating is enabled for the user and gate the entire sequence in that case
+        if self.is_time_limited and self.location.block_type == 'sequential':
```

Comment:
Why does this have to check its own `block_type`? Are there other flavors of `sequential` using this same rendering code that shouldn't be time limited?

Reply:
i think this was left over from when i was trying to do this in the block transformers, i'll remove it